test/fork/Fork.t.sol

```diff
@@ -36,8 +36,8 @@ abstract contract Fork_Test is Base_Test, V2CoreFuzzers {
     //////////////////////////////////////////////////////////////////////////*/
 
     function setUp() public virtual override {
-        // Fork the Goerli testnet.
-        vm.createSelectFork({ blockNumber: 9_261_300, urlOrAlias: "goerli" });
+        // Fork Ethereum Mainnet at a specific block number.
```

Comment:
Let's add the date of the block, as we did in core:

https://github.com/sablier-labs/v2-core/blob/159e87a2f5af03967faf292df81fef93c14be2e2/test/fork/Fork.t.sol#L38

Reply:
I thought about that and decided not to do it because it adds maintenance overhead.

I will remove that date comment in V2 Core

Reply:
https://github.com/sablier-labs/v2-core/commit/98ec942fb001b5846b285ee3679094d05aefc007